src/bp-moderation/classes/class-bp-moderation.php

```diff
@@ -314,14 +314,18 @@ public function save() {
 		}
 
 		// Get Moderation settings.
-		$threshold          = false;
-		$email_notification = false;
+		$threshold                  = false;
+		$email_notification         = false;
+		$content_block_notification = false;
+		$user_block_notification    = false;
```

Comment:
@janakkaneriya007 I already created `email_notification` to check setting why we need an extra variable `user_block_notification`